python3 updates to ezid.py

mburrus opened the pull request:
I made syntax changes where needed and changed package nomenclature (urllib package is used in Python3 and I chose to specify the specific modules needed to execute the script)

Only additional change is in line 242 where I specified that the returned object should be a list.

mburrus committed:
python3 updates to ezid.py

Made syntax changes where needed and changed package nomenclature (urllib2 underwent some changes betwen Python2 and Python3)

Only additional change is in line 242; specified that the returned object should be a list.

rushirajnenuji commented:
Hey @mburrus  - I reviewed the pull request (PR) and noticed that both the PRs are replacing the existing file. EZID would like to retain Python 2 files as well (some clients still use this version of Python). So, could you please either edit your PR (and add the new files `ezid3.py` and `batch_register3.py` while keeping the current files in place)?

I have fixed both these Python3 files and tested them - so from my end, the fix is complete. I can push my changes as soon as you update the PR (or create a new PR - whatever works for you!) Thanks!

mburrus commented:
Hi Rushiraj,

Sure thing, I'll give this a try and update you once I've changed the PR

On Thu, Jan 9, 2020 at 1:11 PM Rushiraj Nenuji <notifications@github.com>
wrote:

> Hey @mburrus <https://github.com/mburrus> - I reviewed the pull request
> (PR) and noticed that both the PRs are replacing the existing file. EZID
> would like to retain Python 2 files as well (some clients still use this
> version of Python). So, could you please either edit your PR (and add the
> new files ezid3.py and batch_register3.py while keeping the current files
> in place)?
>
> I have fixed both these Python3 files and tested them - so from my end,
> the fix is complete. I can push my changes as soon as you update the PR (or
> create a new PR - whatever works for you!) Thanks!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/CDLUC3/ezid-client-tools/pull/1?email_source=notifications&email_token=AM4UHVLDZANQDB7UEEN34OLQ46HJXA5CNFSM4JU5ZY2KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEIRZRSI#issuecomment-572758217>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AM4UHVMRMG4JDO525H2UTKLQ46HJXANCNFSM4JU5ZY2A>
> .
>

mburrus commented:
This Pull Request has been closed. See 'Python3 updates (second pull request)'
